registration/apps.py

```diff
@@ -0,0 +1,6 @@
+from django.apps import AppConfig
+
+
+class AccountsConfig(AppConfig):
```

Comment:
The class name `AccountsConfig` does not match the app name 'registration'. It is recommended to name the class `RegistrationConfig` to avoid confusion and maintain consistency.